data/json/mutations/mutation_spells.json

```diff
@@ -18,5 +18,25 @@
     "flags": [ "SILENT", "NO_LEGS", "NO_HANDS", "RANDOM_DAMAGE", "RANDOM_DURATION", "RANDOM_TARGET" ],
     "energy_source": "NONE",
     "damage_type": "bio"
```

Comment:
Looks like you made this on an older version of the file. Changing the damage type here to "biological" should fix the merge conflict.

Reply:
Bio is for poison effects. It needs to be electric because it's an electric shock.

Reply:
Ah, I'll update MAGIC.md to reflect that.